File: API/Models/Story.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Models
{
    public class Story
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Intro { get; set; }
        public string Content { get; set; }
        public string Slug { get; set; }
        public string ImageUrl { get; set; }
        public DateTime DatePosted { get; set; }
        public int Views { get; set; } = 1;
        public string Category { get; set; }
        public string Author { get; set; }
       
    }
}

File: Spa/src/app/_models/story.ts
import {Image} from "./Image";

export interface Story {
  id:number;
  title?:string;
  intro?:string;
  content?:string;
  imageUrl:string;
  datePosted?:Date;
  category?:string;
  author?:string;
  slug?:string;
  views?:number;


}
